Corona-Warn-App/src/test/java/de/rki/coronawarnapp/ui/eventregistration/organizer/create/TraceLocationCreateViewModelTest.kt

```diff
@@ -18,10 +22,17 @@ import testhelpers.extensions.observeForTesting
 @ExtendWith(InstantExecutorExtension::class)
 internal class TraceLocationCreateViewModelTest : BaseTest() {
 
+    @MockK lateinit var traceLocationCreator: TraceLocationCreator
+
+    @BeforeEach
+    fun setup() {
+        MockKAnnotations.init(this)
+    }
+
     @ParameterizedTest
     @MethodSource("provideArguments")
     fun `send should not be enabled for empty form`(category: TraceLocationCategory) {
-        val viewModel = TraceLocationCreateViewModel(category = category)
+        val viewModel = TraceLocationCreateViewModel(category = category, traceLocationCreator = traceLocationCreator)
```

Comment:
In other tests, we usually have a `createInstance()` function that creates the class we want to test and then call it from every test.

Reply:
updated